includes/Classifai/Providers/Azure/ComputerVision.php

```diff
@@ -160,6 +253,32 @@ public function setup_fields_sections() {
 				'description'   => __( 'Minimum confidence score for automatically applied image captions, numeric value from 0-100. Recommended to be set to at least 75.', 'classifai' ),
 			]
 		);
+		add_settings_field(
+			'enable-image-tagging',
+			esc_html__( 'Automatically Tag Images', 'classifai' ),
+			[ $this, 'render_input' ],
+			$this->get_option_name(),
+			$this->get_option_name(),
+			[
+				'label_for'     => 'enable_image_tagging',
+				'input_type'    => 'checkbox',
+				'default_value' => true,
+				'description'   => __( 'Uploaded images will be auto-tagged', 'classifai' ),
+			]
+		);
+		add_settings_field(
+			'image-tag-threshold',
+			esc_html__( 'Tag Confidence Threshold', 'classifai' ),
+			[ $this, 'render_input' ],
+			$this->get_option_name(),
+			$this->get_option_name(),
+			[
+				'label_for'     => 'tag_threshold',
+				'input_type'    => 'number',
+				'default_value' => 75,
```

Comment:
We'll want to test to see if this threshold makes sense in some different types of images

Reply:
@helen updated to 70 as the default